project1-sps/script.js

```diff
@@ -1,4 +1,141 @@
+// };
+
+// var greetUser = function (name) {
+//   initialMessage();
+//   return `Hello ${name}. Please enter either scissors, paper, or stone to begin.`;
+// };
+
+var main = function (userInput) {
+  //when user inputs anything that is not either the keywords (scissors, paper, or stone), the output will remind them to attempt the correct input again.
+  //only when the user inputs the exact keyword (of either scissors, paper, or stone), the program (inclusive of formula, value assignment, and conditions/rules) will run.
+
+  if (
+    userInput === `scissors` ||
+    userInput === `paper` ||
+    userInput === `stone`
+  ) {
+    //computer randomizes scissors, paper or stone.
+    //formula/computation to compute the integers 1, 2 and 3.
+    var computerRandomize = Math.floor(Math.random() * 3) + 1;
+    console.log(`logging computerRandomize ${computerRandomize}`);
+
+    //assigning each randomized integer (1,2,3) to an item (scissor, paper or stone <- such that user and computer are comparing the same thing).
+    if (computerRandomize === 1) {
+      computerRandomize = `scissors`;
+    } else if (computerRandomize === 2) {
+      computerRandomize = `paper`;
+    } else if (computerRandomize === 3) {
+      computerRandomize = `stone`;
+    }
+  }
+  if (userInput === `scissors` && computerRandomize === `paper`) {
+    playerWinCount += 1;
+    return "You win!" + "<br>" + "<br>" + whoWinMessage();
+  }
+  if (userInput === `scissors` && computerRandomize === `stone`) {
+    computerWinCount += 1;
+    return "You Lose." + "<br>" + "<br>" + whoWinMessage();
+  }
+  if (userInput === `scissors` && computerRandomize === `scissors`) {
+    drawCount += 1;
+    return "Oh, its a draw." + "<br>" + "<br>" + whoWinMessage();
+  }
+  //define what computer outputs when user inputs paper.
+  if (userInput === `paper` && computerRandomize === `paper`) {
+    drawCount += 1;
+    return "Oh, its a draw." + "<br>" + "<br>" + whoWinMessage();
+  }
+  if (userInput === `paper` && computerRandomize === `stone`) {
+    playerWinCount += 1;
+    return "You win!" + "<br>" + "<br>" + whoWinMessage();
+  }
+  if (userInput === `paper` && computerRandomize === `scisors`) {
+    computerWinCount += 1;
+    return "You lose." + "<br>" + "<br>" + whoWinMessage();
+  }
+  //define what computer outputs when user inputs stone.
+  if (userInput === `stone` && computerRandomize === `paper`) {
+    computerWinCount += 1;
+    return "You lose." + "<br>" + "<br>" + whoWinMessage();
+  }
+  if (userInput === `stone` && computerRandomize === `stone`) {
+    drawCount += 1;
+    return (
+      "Oh, its a draw. You can try again." + "<br>" + "<br>" + whoWinMessage()
+    );
+  }
+  if (userInput === `stone` && computerRandomize === `scissors`) {
+    playerWinCount += 1;
+    return (
+      "You win! You can try playing again" + "<br>" + "<br>" + whoWinMessage()
+    );
+  } else
+    return `To play this game, please input either scissors or paper or stone.`;
+};
```

Comment:
I like that you have a function for your output message. It shortens the code here.

Maybe you can try group the winning, losing and draw conditions so you have lesser return statements on your next project!